### PR TITLE
Apply mutation pages PR to 6.x

### PR DESCRIPTION
**Pull request by rsill-neo4j**

### CODE REVIEW

File: modules/ROOT/pages/mutations/create.adoc [hunk 1]
```diff
@@ -152,15 +153,17 @@ mutation {
...
 
-This will ensure that a movie with ID 1234 exists and it is connected to `"Tom Hanks"`. If the movie does not exist, it will be created with the title `"Forrest Gump"`. Note that if the movie with the given ID already exists, it will be connected to it, regardless of the title.
+This ensures that a movie with ID 1234 exists and is connected to `"Tom Hanks"`.
+If the movie does not exist, it will be created with the title `"Forrest Gump"`.
+If a movie with the given ID already exists, it will also be connected to `"Tom Hanks"`, and keep whatever title it has.
 
 == `CREATE` optimization
 
-With the `create` operations, there is no limit on how many nodes can be created at once.
+With `CREATE` operations, there is no limit on how many nodes can be created at once.
```
Comment (MacondoExpress) [inline modules/ROOT/pages/mutations/create.adoc:162]: Nitpick: FYI  the `connectOrCreate` is camelcase but `CREATE` all uppercase idk if this is intended

File: modules/ROOT/pages/mutations/create.adoc [hunk 2]
```diff
@@ -111,7 +112,7 @@ Read about xref:mutations/update.adoc#_connectorcreate_relationships[`update`] f
 
 == `connectOrCreate` relationships
 
-If a related node has the `@unique` directive defined, `connectOrCreate` can be used in a nested `create` to perform a `MERGE` operation on the related node.
+If a related node has the `@unique` directive defined, you can use `connectOrCreate` nested in a `CREATE` mutation to perform an operation similar to a link:https://neo4j.com/docs/cypher-manual/current/clauses/merge/[Cypher `MERGE`] operation on the related node.
```
Comment (MacondoExpress) [inline modules/ROOT/pages/mutations/create.adoc]: I'll ping @angrykoala for this rephrase as he knows better

File: modules/ROOT/pages/mutations/create.adoc (hunk 1 above)
Comment (MacondoExpress) [inline modules/ROOT/pages/mutations/create.adoc]: Nitpick: FYI  the `connectOrCreate` is written camelcase above but `CREATE` all uppercase idk if this is intended

File: modules/ROOT/pages/mutations/delete.adoc [hunk 3]
```diff
@@ -74,8 +74,7 @@ mutation {
 }
 ----
 
-By the time the traversal has reached it, that empty `where` argument has the context of only refer to posts that were created by Jane Doe, as the traversals to those `Post` nodes were from her `User` node. 
-Essentially, the above query is equivalent to:
+This query is equivalent to:
```
Comment (MacondoExpress) [inline modules/ROOT/pages/mutations/delete.adoc]: TBF, I believe this last example add more confusion that benefit, not a as part of you're PR but I would delete this part entirely, I'll ping @mjfwebb and @angrykoala for a couple of extra eyes here.

**Review by MacondoExpress — COMMENTED** [review]
Lovely, I have pinged other team member in two comments where I'm not 100% confident, but apart from that it looks good to me!

File: modules/ROOT/pages/mutations/delete.adoc (hunk 3 above)
Comment (mjfwebb) [reply modules/ROOT/pages/mutations/delete.adoc]: Yes I agree, after a small discussion we can just get rid of this last part.

Comment (rsill-neo4j) [reply modules/ROOT/pages/mutations/delete.adoc]: probably relates to what @mjfwebb posted in the docs-graphql channel: https://neo4j.slack.com/archives/C03E32T5MCL/p1729502123351959

so, get rid of the whole example then?

File: modules/ROOT/pages/mutations/create.adoc (hunk 1 above)
Comment (rsill-neo4j) [reply modules/ROOT/pages/mutations/create.adoc]: nice catch. i'll revisit those (it's in the - untouched - heading too, line 160)

File: modules/ROOT/pages/mutations/delete.adoc (hunk 3 above)
Comment (MacondoExpress) [reply modules/ROOT/pages/mutations/delete.adoc]: In my opinion yes, I would get rid of the `This query is equivalent to:`  with the following example

Comment (rsill-neo4j) [reply modules/ROOT/pages/mutations/delete.adoc]: done

**Review by MacondoExpress — APPROVED** [review]

File: modules/ROOT/pages/mutations/create.adoc (hunk 2 above)
Comment (angrykoala) [reply modules/ROOT/pages/mutations/create.adoc]: This seems accurate enough

**Pull request merged**